server/controllers/aidControllers.js

```diff
@@ -0,0 +1,96 @@
+import { INTERNAL_SERVER_ERROR,CREATED,OK,NOT_FOUND,BAD_REQUEST } from 'http-status-codes';
+const db = require("./promise").AidDb;
+import validateAidQueryText from '../validation/aid'
+
+
+const Aids = {
+    async create(req, res) {
+        const { errors, isValid } = validateAidQueryText(req.body);
+
+		// Check Validation
+		if (!isValid) {
+			return res.status(BAD_REQUEST).json(errors);
+		}
+        const queryText = req.body;
+        try {
+          const createdAid = await db.create(queryText);
+          return res.status(CREATED).send({
+            message: 'Aid successfully created',
+            data: createdAid,
+          });
```

Comment:
Use JSend specification (https://github.com/omniti-labs/jsend) when sending out of your endpoints
```
...
const aid = await db.create(queryText);
return res.status(CREATED).send({
    status: 'success',
    data: { aid, message: 'Aid successfully created' },
});
...